libs/live-nft/src/supported.ts

```diff
@@ -1,62 +1,28 @@
 /**
  * Supported blockchains EVM by Backend
  */
-export enum BlockchainEVM {
+export enum Blockchain {
   Arbitrum = "arbitrum",
   Avalanche = "avalanche_c_chain",
   Base = "base",
-  Blast = "blast",
   Bsc = "bsc",
-  Canto = "canto",
-  Celo = "celo",
-  Cyber = "cyber",
-  Degen = "degen",
   Ethereum = "ethereum",
-  Fantom = "fantom",
-  Gnosis = "gnosis",
-  Godwoken = "godwoken",
-  Linea = "linea",
-  Loot = "loot",
-  Manta = "manta",
-  Mode = "mode",
-  Moonbeam = "moonbeam",
-  Opbnb = "opbnb",
   Optimism = "optimism",
-  Palm = "palm",
   Polygon = "polygon",
-  ProofOfPlay = "proofofplay",
-  Rari = "rari",
-  Scroll = "scroll",
-  Sei = "sei_network",
-  Xai = "xai",
-  Zora = "zora",
+  Solana = "solana",
 }
 
-export const blockchainEVMList: BlockchainEVM[] = Object.values(BlockchainEVM);
-
 export const SUPPORTED_BLOCKCHAINS_LIVE = [
-  // BlockchainEVM.Arbitrum,
-  BlockchainEVM.Avalanche,
-  //BlockchainEVM.Base,
-  // BlockchainEVM.Blast,
-  BlockchainEVM.Bsc,
-  // BlockchainEVM.Celo,
-  BlockchainEVM.Ethereum,
-  // BlockchainEVM.Fantom,
-  // BlockchainEVM.Linea,
-  // BlockchainEVM.Moonbeam,
-  //BlockchainEVM.Optimism,
-  BlockchainEVM.Polygon,
-  //BlockchainEVM.Scroll,
-  // BlockchainEVM.Sei,
+  Blockchain.Avalanche,
+  Blockchain.Bsc,
+  Blockchain.Ethereum,
+  Blockchain.Polygon,
+  Blockchain.Solana,
 ];
```

Comment:
Typescript infer `SUPPORTED_BLOCKCHAINS_LIVE: Blockchain[]`
As a result there is no difference between the types `Blockchain` and `SupportedBlockchains`:

<img width="758" alt="image" src="https://github.com/user-attachments/assets/642177a7-ab4e-4979-b9ba-bb56aef598db" />

```suggestion
] as const;
```

Fixes this by making:

<img width="758" alt="image" src="https://github.com/user-attachments/assets/40b75885-3c7a-42f2-b8b5-cf4187b0f892" />